mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -166,6 +166,9 @@ class LinearRegression @Since("1.3.0") (@Since("1.3.0") override val uid: String
    * The default value is "auto" which means that the solver algorithm is
    * selected automatically.
    *
+   * The Normal Equation solver is limited to a few thousand features; when needed,
```

Comment:
Is this a bit misleading? If I set `solver = "normal"` and my dataset contains >4096 features, an exception is thrown instead of automatically falling back to iterative methods.